go/keybase/main.go

```diff
@@ -46,8 +46,19 @@ func keybaseExit(exitCode int) {
 	os.Exit(exitCode)
 }
 
+// Preserve non-critical errors that happen very early during startup,
+// where logging is not set up yet, to be printed later when logging
+// is functioning.
+var startupErrors []error
```

Comment:
Can we make this not a global, and just pass it down into `mainInner`?

Reply:
sounds great. my first attempt was having this in `G` which was an overkill, so I missed the simplest solution